server.go

```diff
@@ -2492,7 +2492,11 @@ func (s *server) Start() error {
 		// brontide.NewListener() is called in newServer. This means
 		// that we are actually listening and partially accepting
 		// inbound connections even before the connMgr starts.
+		//
+		// TODO(yy): move the log into the connMgr's `Start` method.
```

Comment:
nit: instead of adding the TODO, why not just move it?

Reply:
Would need to update the `btcd` package, then a few more updates...so this PR is created to debug #9629, kinda wanna the user to patch this and give us the log so we can fix it in 0.19